x-pack/solutions/security/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_list.test.tsx

```diff
@@ -61,9 +61,10 @@ describe('When on the trusted applications page', () => {
 
   it('should search using expected exception item fields', async () => {
     const expectedFilterString = parseQueryFilterToKQL('fooFooFoo', SEARCHABLE_FIELDS);
-    const { findAllByTestId } = render();
+    const { getAllByTestId } = render();
+
     await waitFor(async () => {
-      await expect(findAllByTestId('trustedAppsListPage-card')).resolves.toHaveLength(10);
+      expect(getAllByTestId('trustedAppsListPage-card')).toHaveLength(10);
```

Comment:
fixes [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/267663#01946e9c-d644-4c6f-a1e3-ab4a093833c5) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/267663/jobs/01946e9c-d644-4c6f-a1e3-ab4a093833c5/artifacts/01946ec3-4184-4b47-9011-ac9315070264) Jest Tests / When on the trusted applications page should search using expected exception item fields